src/components/AppLayout/AccountSwitcher.tsx

```diff
@@ -58,19 +59,37 @@ const ActionButtons = ({ logout, close }: { logout: () => Promise<void>; close:
       });
   };
 
+  const handleLogout = () => {
+    setLoggingOut(true);
+    logout().then();
+  };
+  const handleLogoutAll = () => {
+    setLoggingOutAll(true);
+    logoutAll().then();
+  };
```

Comment:
I'd say we can use the `.then()` to reset each state, just in case. We can probably use finally instead so it sets to false even if it fails, plus we avoid the promise warning 😛 
```suggestion
  const handleLogout = () => {
    setLoggingOut(true);
    logout().finally(() => setLoggingOut(false));
  };
  const handleLogoutAll = () => {
    setLoggingOutAll(true);
    logoutAll().finally(() => etLogginOutAll(false));
  };
```

Reply:
I tried that, but for some unknown reason it resolves instantly. I can put it in a catch and then raise e again. It doesn't matter for success because the page refreshes.